src/js/apps/forms/form/form_app.js

```diff
@@ -220,9 +236,10 @@ export default App.extend({
         'submit'() {
           this.showForm();
         },
-        'cancel'() {
+        'discard:submission'() {
           Radio.request(`form${ this.form.id }`, 'clear:storedSubmission');
           this.showForm();
+          this.showLastUpdated();
```

Comment:
Also just added this in the last push.

It removes the `Last edit was ...` text when the user discards their stored submission.